wetterdienst/core/scalar/request.py

```diff
@@ -349,6 +349,7 @@ def __init__(
         period: Period,
         start_date: Optional[Union[str, datetime, pd.Timestamp]] = None,
         end_date: Optional[Union[str, datetime, pd.Timestamp]] = None,
+        settings: Settings = Settings.default(),
```

Comment:
Please don't do that. Instead, change the signature like this,
```suggestion
        settings: Settings = None,
```

and then, within the body of the method, do
```
settings = settings or Settings.default()
```

Reply:
lint would not have allowed me anyway

Reply:
Did it yell s.th. about _mutable defaults_?

-- https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/